renovate.json

```diff
@@ -9,6 +9,18 @@
     "postUpdateOptions": [
         "gomodTidy"
     ],
+    "customManagers": [
```

Comment:
@johanbrandhorst, I added this configuration following what I saw in [the documentation](https://docs.renovatebot.com/modules/manager/regex/) to keep the Bazel version up to date in the `presubmit.yml` file.

I don't have experience with _renovate_.

Reply:
Hm, I haven't ever used this either, so I have no idea if this will work. Is there any way we can test it?